python/pyarrow/tests/interchange/test_conversion.py

```diff
@@ -194,18 +194,19 @@ def test_roundtrip_pandas_string():
     if Version(pd.__version__) < Version("1.6"):
         pytest.skip(" Column.size() called as a method in pandas 2.0.0")
 
-    # large string is not supported by pandas implementation
-    table = pa.table({"a": pa.array(["a", "", "c"])})
+    arr = ["a", "", "c"]
```

Comment:
One more thing: we can maybe add a `None` here to also test missing values support?